components/brave_wallet/browser/solana_tx_manager.cc

```diff
@@ -361,6 +366,39 @@ void SolanaTxManager::MakeTokenProgramTransferTxData(
           std::move(callback)));
 }
 
+void SolanaTxManager::MakeTokenProgramTxDataFromMessage(
```

Comment:
This should probably be named as `MakeTxDataFromBase64EncodedMessage`.

Reply:
Renamed to `MakeTxDataFromBase64EncodedTransaction`.